README.md

```diff
@@ -1,45 +1,231 @@
+| [Library](operator/pkg)                                   | In active development - expect bugs and fast paced development. Detailed documentation to follow. |
+
+## Operator specification
+
+The module-manager reconciles the `Manifest` custom resource.
+As observed in the [Sample Manifest CR](operator/config/samples/operator_v1alpha1_manifest.yaml) and [API definition](operator/api/v1alpha1/manifest_types.go), 
+the `Spec` contains the necessary properties to process installations.
+
+### Manifest Custom Resource
+
+| Spec field | Description                                                                                                   |
+|------------|---------------------------------------------------------------------------------------------------------------|
+| Remote     | `false` for single cluster mode, `true`(default) for dual cluster mode                                        |
+| Resource   | Additional unstructured custom resource to be installed, e.g. for implicit reconciliation via Installs        |
+| Installs   | OCI image specification for a list of helm charts                                                             |
+| Config     | Optional: OCI image specification for helm configuration and set flags                                        |
+| CRDs       | Optional: OCI image specification for additional CRDs that are pre-installed before helm charts are processed |
+
+If `.Spec.Remote.` is set to `true`, the operator will look for a secret with the name specified by Manifest CR's label `operator.kyma-project.io/kyma-name: kyma-sample`.
+Follow the steps in [this guide](https://github.com/kyma-project/lifecycle-manager/blob/main/docs/developer/creating-test-environment.md#install-kyma-and-run-lifecycle-manager-operator) to create the required secret.
+This secret is used to connect to an existing cluster (target) for `Manifest` resource installations.
+
+For more details on OCI Image **bundling** and **formats**, refer to our [bundling and installation guide](https://github.com/kyma-project/lifecycle-manager/tree/main/samples/template-operator#bundling-and-installation).
+The component-descriptor generated from this guide could be used to independently build a `Manifest Spec` based on the OCI image specifications.
+
+>Note: [Lifecycle-Manager](https://github.com/kyma-project/lifecycle-manager#how-it-works) translates these layers from a `ModuleTemplate` resource on the Kyma control-plane and translates them automatically to a subsequent `Manifest` resource.
+>Alternatively you could use your own bundled OCI images. Please take care to conform to [.Spec.Config](https://github.com/kyma-project/lifecycle-manager/blob/main/samples/template-operator/config.yaml) format,
+> which corresponds to helm configuration and set value flags for an installation in `.Spec.Installs[].Name`.
+
+### Sample Resource
```

Comment:
We use [sentence case](https://kyma-project.io/community/guidelines/content-guidelines/02-style-and-terminology/#sentence-case) in headlines. 

```suggestion
### Sample resource
```